bindings/go/examples/pagination/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+
+	sdk "bindings/iota_sdk_ffi"
+)
+
+func main() {
+	client := sdk.GraphQlClientNewDevnet()
+	address, err := sdk.AddressFromHex("0x611830d3641a68f94a690dcc25d1f4b0dac948325ac18f6dd32564371735f32c")
+	if err != nil {
+		panic(err)
```

Comment:
I think in the other examples we use `log.Fatalf`. Wondering if we should do the same to be consistent?